Documentation/blobIn.md

```diff
@@ -1,3 +1,12 @@
+#### Connecting to your Storage Account
+
+* Download Storage explorer from here: http://storageexplorer.com/ 
+* Connect using these credentials:
+* Account Name: `{{accountName}}`
+* Account Key: `{{accountKey}}`
```

Comment:
I'll close this PR. The new change to hide secrets by default will directly be made in the AzureFunctionsPortal, and wont need any changes here.
Adding custom html on the fly is apparently not supported by angular.